client/registry_login.go

```diff
@@ -0,0 +1,22 @@
+package client
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/alibaba/pouch/apis/types"
+)
+
+// RegistryLogin requests a registry server to login.
```

Comment:
This comment is quite confusing. Could we change this?

Reply:
Sure.